cms/djangoapps/contentstore/views/library.py

```diff
@@ -115,9 +115,6 @@ def _create_library(request):
     """
     Helper method for creating a new library.
     """
-    if not auth.has_access(request.user, CourseCreatorRole()):
```

Comment:
Rather than removing the auth check altogether, how about we create a specific check for libraries and encapsulate any rules inside -- for example, we would at least want to ensure the feature is enabled.